Complete WelcomeView

theheleo opened the pull request:
Полностью закончил верстку WelcomeView и удалил у себя случайно запулленную ветку Арсентия, так что можно мержить с develop :)

Hieronius committed:
test commit

Hieronius committed:
Add elements and constraints to Result View Controller

Commit:
WelcomeView is finished

Commit:
Merge branch 'features/ResultModule' of https://github.com/SiLVERSHTAiN/Who-wanna-be-milionaire into feature/WelcomeModule

Commit:
Удалил у себя случайно запуллинный коммит Арсентия

ghost commented:
![image](https://user-images.githubusercontent.com/78638629/217381853-b576cc56-cd42-4673-98d5-dfa90447788d.png)

SiLVERSHTAiN commented on Who-wanna-be-millionaire/Modules/WelcomeModule/Controller/WelcomeViewController.swift:
```diff
@@ -1,35 +1,57 @@
     
+    func setupViews() {
+        
+        view.addSubview(backgroundImage)
```
Привет, чтобы не дублировать код у нас есть метод в ext

theheleo replied:
Из-за того, что я изменил backgroundImage во WelcomeViewController, там класс наследуется от UIViewController, а метод в расширении для UIView. Мне добавить такой же метод в ext для UIViewController?

SiLVERSHTAiN commented on Who-wanna-be-millionaire/Modules/WelcomeModule/View/ButtonsView.swift:
```diff
@@ -0,0 +1,61 @@
+    private lazy var roolsButton: UIButton = {
+        let button = UIButton(type: .system)
+        let buttonTitleStr = NSMutableAttributedString(string: WelcomeConstants.roolsButtonTitle, attributes:attrs)
+        roolsAttributedString.append(buttonTitleStr)
```
Ты можешь на прямую обращаться в свойству text в кнопке

theheleo replied:
Да, я знаю, но это единственный способ, который нашёл, чтобы текст подчеркнутым был

SiLVERSHTAiN reviewed (commented):
Добавил коменты, что надо исправить

Commit:
Fix bugs

1. Переструктурировал энум WelcomeConstant, добавил метод setFont.
2. Удалил лишние строки.
3. Устранил мелкие недочёты :)

Commit:
Fix WelcomeView Bugs 2

Сделал view.safeAreaLayoutGuide.topAnchor и максимально укоротил функцию
